projects/storefrontapp-e2e-cypress/cypress.config.ts

```diff
@@ -23,6 +23,7 @@ export default defineConfig({
     OCC_PREFIX: '/occ/v2',
     OCC_PREFIX_USER_ENDPOINT: 'users',
     OCC_PREFIX_ORDER_ENDPOINT: 'orders',
+    RECAPTCHA_PUBLIC_KEY: '6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI',
```

Comment:
Is having this key in code approved by the security team? I'm not sure another easy way to go about it in local testing but I'd like to cross off any concern they may have.

Reply:
Just confirm with security guy, the answer is not. Seems we need to find another way to enable captcha for test. Do you have any other ideas?

Reply:
Yes, it should be possible to add a secret key in the ci workflow (our github actions) using github secrets. I believe you will need to speak to an admin to do this.